stdlib/3/subprocess.pyi

```diff
@@ -8,14 +8,14 @@ from types import TracebackType
 
 if sys.version_info >= (3, 5):
     class CompletedProcess:
-        args = ...  # type: Union[List, str]
+        args = ...  # type: Union[Sequence[str], str]
         returncode = ...  # type: int
-        stdout = ...  # type: Any
-        stderr = ...  # type: Any
+        stdout = ...  # type: Union[str, bytes, None]
+        stderr = ...  # type: Union[str, bytes, None]
```

Comment:
I see, that makes sense. Another option here might be to make CompletedProcess generic over AnyStr, so that we can at least ensure that the type passed to the constructor is the same as the type of the attribute. However, I can't see a lot of scenarios in which that would actually be helpful in code, so I'll just change these back to Any.